tfhe/Cargo.toml

```diff
@@ -100,12 +100,15 @@ boolean-c-api = ["boolean", "__c_api"]
 shortint-c-api = ["shortint", "__c_api"]
 high-level-c-api = ["boolean-c-api", "shortint-c-api", "integer", "__c_api"]
 
+wasm = [
+    "dep:getrandom",
+]
```

Comment:
Use a traget_arch style dependency with `target_arch = "wasm32"`

see e.g. https://github.com/zama-ai/concrete/blob/359d4c59190f0f0cb70ec6b6dba77b1c24dfea76/concrete-shortint/Cargo.toml#L23

Reply:
This is required to have `getrandom` as a dependency for the wasm build

Reply:
Yes, you are correct! It looks like during the rebase I lost that change, I will add it back in the squashed commit

Reply:
hey @PedroMBB looks like the pre commit check failed for the build

https://github.com/zama-ai/tfhe-rs/actions/runs/7627525779/job/20776459136?pr=783#step:4:349

For the Cargo.toml I think you need

```toml
# Remove this line
getrandom = { version = "0.2.8", optional = true }
bytemuck = "1.13.1"

# add
[target.'cfg(target_arch = "wasm32")'.dependencies.getrandom]
 version = "0.2.8"
```

and remove the dep:getrandom from the __wasm_api feature

if you want to make sure the check passes you can run make pcc locally before pushing

Reply:
ah wait that's something a bit different... hmm

is it ok if we use your current patch and give co-authorship and we can look at getting this in the code base ?

Reply:
otherwise I think we'll go for your first solution as it seems our CI tooling is not ready to handle the proper way of doing things (sorry) and I don't want to go for a lot of rounds getting the CI fixed, let me know if you just want to revert to the first version of this

Reply:
> ah wait that's something a bit different... hmm
> 
> is it ok if we use your current patch and give co-authorship and we can look at getting this in the code base ?

Yes, it is okay for me. Do you need me to do anything for that to happen?

Reply:
no we will add you as a co-author on the commit and we will tag you on the PR :)